### PR TITLE
add blog

### PR DESCRIPTION
**Pull request by RominaFior**

### CODE REVIEW

**Comment by alexgbor** [comment]
Todo correcto Romina!! 👍

File: staff/RominaFior/blog/index.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="icon" href="images/broccoli.png" type="image/x-icon" sizes="16x16">
+    <title>Blog</title>
+</head>
+<header>
```
Comment (albanelia0) [inline staff/RominaFior/blog/index.html]: this should be inside body! (;

Comment (albanelia0) [reply staff/RominaFior/blog/index.html]: but very well!!

Comment (RominaFior) [reply staff/RominaFior/blog/index.html]: Done 👍

File: staff/RominaFior/blog/index.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            <ul><li>Cosmética</li></ul>
+            <li>2017</li>
+            <ul><li>Yoga</li></ul>
+            <li>2018</li>
+            <ul><li>Restaurantes veganos Barcelona</li></ul>
+        </ul>
+
+    </aside>
+    <footer>
+        <p>© 2018 ROMINA FIOR ALL RIGHTS RESERVED</p>
```
Comment (manuelbarzi) [inline staff/RominaFior/blog/index.html]: WARN! for special symbols, like copyright, use always html entities. in this case: 

```&copy;```

SEE https://www.w3schools.com/html/html_entities.asp for more details

File: staff/RominaFior/blog/index.html [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    </main>
+    <aside>
+        BUSCAR ESTE BLOG
+        <br>
+        <input type="text" value="" placeholder="">
+        <button>Buscar</button>
+        <br> SEGUIDORES
+        <br>
+        <input type="text" value="" placeholder="">
+        <button>Seguir</button>
```
Comment (manuelbarzi) [inline staff/RominaFior/blog/index.html]: WRAP input and button inside a form tag. 

SEE https://www.w3schools.com/html/html_forms.asp

**Comment by albanelia0** [comment]
Nice structure!! ^^

File: staff/RominaFior/blog/index.html (hunk 3 above)
Comment (RominaFior) [reply staff/RominaFior/blog/index.html]: done!

File: staff/RominaFior/blog/index.html (hunk 2 above)
Comment (RominaFior) [reply staff/RominaFior/blog/index.html]: done!

**Pull request merged**